app/src/main/java/com/example/recyclerviewdemo/Person.java

```diff
@@ -0,0 +1,29 @@
+package com.example.recyclerviewdemo;
+
+public class Person {
+    private int mImage;
+    private String mTen;
+
+    public Person(int mImage, String mTen) {
```

Comment:
```suggestion
    public Person(int image, String name) {
```
tương tự ở những nơi khác